searx/webapp.py

```diff
@@ -39,8 +39,7 @@
     from pygments.formatters import HtmlFormatter
 except:
     logger.critical("cannot import dependency: pygments")
-    from sys import exit
-    exit(1)
+    sys.exit(1)
```

Comment:
Does this code fix / enhance / modify the plugin system ?

We can just remove try...except, but this is another PR in my opinion.

Reply:
> Does this code fix / enhance / modify the plugin system ?

No, but I fixed it BTW.  The sys modul was already imported some [lines above](https://github.com/asciimoo/searx/blob/65236deb9cc255912267983546a9599ca155e1a3/searx/webapp.py#L29), no need to import the name 'exit' explicit .. or did I misunderstood your question?

Reply:
Actually about this one, I don't know why there is try...except..., why not let Python crash ?

I wish there are two PR:
* one with all small fixes --> this is one can be reviewed then merged quickly.
* one about the plugins / themes / resources with a smaller diff --> no need to understand that some changes are not related.

Reply:
You are right, my fail .. I have to learn not to fix things BTW.  Sorry, for the future I will improve myself. Could we leave it that way in this PR?